aries-site/src/pages/[topic].js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Heading, Paragraph } from 'grommet';
+
+import { CardGrid, Meta } from '../components';
+import { Layout, PageIntro } from '../layouts';
+import { getCards, getPageDetails, titleCase } from '../utils';
+
+const NavPage = ({ topic: topicProp }) => {
+  const topic = titleCase(topicProp);
+  const pageDetails = getPageDetails(topic);
+  const cards = getCards(topic);
+  React.useEffect(() => console.log(pageDetails), [pageDetails]);
+  return (
+    <Layout
+      title={topic}
+      isLanding
+      backgroundImage={pageDetails.backgroundImage}
+    >
+      <Meta
+        title={topic}
+        description={pageDetails.seoDescription}
+        canonicalUrl={`https://design-system.hpe.design/${topicProp}`}
+      />
+      <Box gap="large">
+        <PageIntro>
+          <Box justify="center" fill>
+            <Heading margin="none">{topic}</Heading>
+            <Paragraph fill>{pageDetails.description}</Paragraph>
+          </Box>
+        </PageIntro>
+        <CardGrid cards={cards} />
+      </Box>
+    </Layout>
+  );
+};
+
+// This function gets called at build time
```

Comment:
Worth including this url in the comment?
https://nextjs.org/docs/basic-features/pages#scenario-2-your-page-paths-depend-on-external-data

Reply:
Done.